cluster/examples/kubernetes/ceph/admission-controller/deployment/webhook.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
What about merging all `.yaml` for admission controller into this one?

Reply:
I thought it would be best to have the component files separated as some of the files are being modified in the script logic before being deployed. Let me know if a single would be a better approach ?

Reply:
Currently, we have a single file called `common.yaml` which has everything we need to create pre-requisite resources for the operator so I'd suggest we follow the same pattern. Also, for users, it's less file to inject.

Reply:
added all resources in webhook.yaml